<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoDialog</class>
 <widget class="QDialog" name="InfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>华文行楷</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>账户编号</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditAccount">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>华文行楷</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>登录密码</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditPassword">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>华文行楷</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>账户昵称</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditName">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>华文行楷</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>身份证号</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditIdentification">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>华文行楷</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>出生时间</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEditBirth">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>华文行楷</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>申请时间</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEditRegister">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>华文行楷</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>所在城市</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEditPlace">
       <property name="font">
        <font>
         <family>华文楷体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonUpdate">
       <property name="font">
        <font>
         <family>华文行楷</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>更新信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="font">
        <font>
         <family>华文行楷</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>取消更新</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
